src/api/contributionsApi.ts

```diff
@@ -24,23 +18,26 @@ export const fetchDefaultEpicContent = async (): Promise<DefaultEpicContent> =>
     const data = await response.json();
     const control = data?.sheets?.control;
 
+    const emptyVariant = {
+        paragraphs: [],
+        showTicker: false,
+        name: 'remote_epic_test',
+    };
+
     const transformedData = control.reduce(
-        (
-            acc: DefaultEpicContent,
-            item: { heading: string; paragraphs: string; highlightedText: string },
-        ) => {
+        (acc: Variant, item: { heading: string; paragraphs: string; highlightedText: string }) => {
             if (!acc.heading && item.heading) {
                 acc.heading = item.heading;
             }
             if (item.paragraphs) {
                 acc.paragraphs.push(item.paragraphs);
             }
             if (item.highlightedText) {
-                acc.highlighted.push(item.highlightedText);
+                acc.highlightedText = acc.highlightedText || item.highlightedText;
```

Comment:
For the default epic right now there's only ever one line of highlighted text, even though the value returned by the API is an array.